src/Cake.Common.Tests/Fixtures/Build/ContinuaCIInfoFixture.cs

```diff
@@ -0,0 +1,95 @@
+using Cake.Common.Build.ContinuaCI.Data;
```

Comment:
Mixed tabs and spaces in this file. Change to spaces and re-indent file (`CTRL+K+D` in Visual Studio).